src/styles/index.scss

```diff
@@ -1,3 +1,72 @@
+$day-hight: 100px;
```

Comment:
There is a typo in the variable name `$day-hight`. It should be `$day-height` to correctly represent the height of the day elements.